docs/sources/mimir/configure/configure-spread-minimizing-tokens/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-spread-minimizing-tokens/
+description: Learn how to migrate ingesters to spread-minimizing tokens.
+weight: 115
```

Comment:
I added a page weight to control where this page appears in the left table of contents. With this change, Spread-minimizing tokens appears after Zone-aware replication. 
![image](https://github.com/grafana/mimir/assets/104772500/4c4c624f-f886-4ad6-a014-d9e268f57124)